client/src/containers/analysis-visualization/analysis-filters/years/component.tsx

```diff
@@ -1,224 +1,59 @@
-import React, { useCallback, useEffect, useState, useMemo, Fragment } from 'react';
-import { toNumber, isFinite } from 'lodash';
-import { Transition } from '@headlessui/react';
-import { ChevronDownIcon, ChevronUpIcon } from '@heroicons/react/solid';
+import React, { useEffect, useMemo } from 'react';
 
 import { useAppDispatch, useAppSelector } from 'store/hooks';
 import { analysis, setFilter } from 'store/features/analysis';
 
 import { useYears } from 'hooks/years';
 
+import YearsRangeFilter, { useYearsRange } from 'containers/filters/years-range';
 import Select from 'components/select';
 
-import type { SelectProps, SelectOptions, SelectOption } from 'components/select/types';
-
 const YearsFilter: React.FC = () => {
   const dispatch = useAppDispatch();
-  const [isOpen, setIsOpen] = useState<boolean>(false);
+
   const { visualizationMode, filters, layer } = useAppSelector(analysis);
-  const { startYear, endYear, materials, indicator } = filters;
-  const [additionalYear, setAdditionalYear] = useState<number>(null);
-  const { data, isLoading } = useYears(layer, materials, indicator);
-  const dataWithAdditionalYear: number[] = useMemo(() => {
-    const result = [...(data || [])];
-    if (additionalYear) result.push(additionalYear);
-    return result;
-  }, [data, additionalYear]);
-  const handleAdditionalYear: SelectProps['onSearch'] = useCallback(
```

Comment:
I didn't include this, as I didn't understand what it does. What is the goal of the additional year?

Reply:
We want to allow the user to choose a year that is not in the database because the app is able to calculate new years. But this is only needed in Analysis. For data and admin purposes it doesn't make any sense.

Reply:
So, we will need it but only for Analysis. Could you isolate this functionality to be applied only in Analysis?

Reply:
@davidsingal I've restored the behavior :)